modules/porous_flow/test/tests/fluidstate/water_vapor_tab.i

```diff
@@ -0,0 +1,316 @@
+  []
+  [pressure_gas]
+    type = PorousFlowPropertyAux
+    variable = pressure_gas
+    property = pressure
+    phase = 1
+    execute_on = 'initial timestep_end'
+  []
+  [saturation_water]
+    type = PorousFlowPropertyAux
+    variable = saturation_water
+    property = saturation
+    phase = 0
+    execute_on = 'initial timestep_end'
+  []
+  [saturation_gas]
+    type = PorousFlowPropertyAux
+    variable = saturation_gas
+    property = saturation
+    phase = 1
+    execute_on = 'initial timestep_end'
+  []
+  [density_water]
+    type = PorousFlowPropertyAux
+    variable = density_water
+    property = density
+    phase = 0
+    execute_on = 'initial timestep_end'
+  []
+  [density_gas]
+    type = PorousFlowPropertyAux
+    variable = density_gas
+    property = density
+    phase = 1
+    execute_on = 'initial timestep_end'
+  []
+  [viscosity_water]
+    type = PorousFlowPropertyAux
+    variable = viscosity_water
+    property = viscosity
+    phase = 0
+    execute_on = 'initial timestep_end'
+  []
+  [viscosity_gas]
+    type = PorousFlowPropertyAux
+    variable = viscosity_gas
+    property = viscosity
+    phase = 1
+    execute_on = 'initial timestep_end'
+  []
+  [temperature]
+    type = PorousFlowPropertyAux
+    variable = temperature
+    property = temperature
+    execute_on = 'initial timestep_end'
+  []
+[]
+
+[Kernels]
+  [mass]
+    type = PorousFlowMassTimeDerivative
+    variable = pliq
+  []
+  [heat]
+    type = PorousFlowEnergyTimeDerivative
+    variable = h
+  []
+[]
+
+[UserObjects]
+  [dictator]
+    type = PorousFlowDictator
+    porous_flow_vars = 'pliq h'
+    number_fluid_phases = 2
+    number_fluid_components = 1
+  []
+  [pc]
+    type = PorousFlowCapillaryPressureBC
+    pe = 1e5
+    lambda = 2
+    pc_max = 1e6
+  []
+  [fs]
+    type = PorousFlowWaterVapor
+    water_fp = water
+    capillary_pressure = pc
+  []
+[]
+
+[FluidProperties]
+
+  [water_true]
+    type = Water97FluidProperties
+  []
+  [water]
+    type = TabulatedBicubicFluidProperties
+    fp = water_true
+    allow_fp_and_tabulation = true
+    fluid_property_file = fluid_properties_extended.csv
+  []
+[]
+
+[Materials]
+  [watervapor]
+    type = PorousFlowFluidStateSingleComponent
+    porepressure = pliq
+    enthalpy = h
+    temperature_unit = Kelvin
+    capillary_pressure = pc
+    fluid_state = fs
+  []
+  [permeability]
+    type = PorousFlowPermeabilityConst
+    permeability = '1e-13 0 0 0 1e-13 0 0 0 1e-13'
+  []
+  [relperm0]
+    type = PorousFlowRelativePermeabilityCorey
+    n = 2
+    phase = 0
+  []
+  [relperm1]
+    type = PorousFlowRelativePermeabilityCorey
+    n = 3
+    phase = 1
+  []
+  [porosity]
+    type = PorousFlowPorosityConst
+    porosity = 0.1
+  []
+  [internal_energy]
+    type = PorousFlowMatrixInternalEnergy
+    density = 2500
+    specific_heat_capacity = 1200
+  []
+[]
+
+[Executioner]
+  type = Transient
+  solve_type = NEWTON
+  end_time = 1
+  nl_abs_tol = 1e-12
+[]
```

Comment:
```suggestion
[Problem]
  solve = false
[]

[Executioner]
  type = Steady
[]
```

Reply:
And then update your gold file: it should have only one line instead of two now.

Reply:
still has two
I think we'd need to pass some execute_on to the csv to prevent any other execution?

Reply:
Yeah, I think it'll need `execute_on = INITIAL` in the `Outputs` block.

Reply:
ok i ll add it

Reply:
done